docs/solarsystem/pds/pds.rst

```diff
@@ -0,0 +1,116 @@
+.. _astroquery.solarsystem.pds:
+
+***********************************************************************************
+PDS Planetary Ring Node Queries (`astroquery.solarsystem.pds`)
+***********************************************************************************
+
+Overview
+========
+
+
+The :class:`~astroquery.solarsystem.pds.RingNodeClass` provides an
+interface to the ephemeris tools provided by the `NASA Planetary Data System's Ring Node System`_ hosted by SETI institute.
+
+Ephemeris
+-----------
+
+In order to query information for a specific Solar System body, a
+``RingNode`` object is instantiated and the :meth:`~astroquery.solarsystem.pds.RingNodeClass.ephemeris` method is called. The following example queries the
+ephemerides of the rings and small moons around Uranus as viewed from ALMA:
+
+.. code-block:: python 
+
+   >>> from astroquery.solarsystem.pds import RingNode
+   >>> import astropy.units as u
+   >>> bodytable, ringtable = RingNode.ephemeris(planet='Uranus',
+   ...                 epoch='2024-05-08 22:39',
+   ...                 location = (-67.755 * u.deg, -23.029 * u.deg, 5000 * u.m)) # doctest: +REMOTE_DATA
+   >>> print(ringtable) # doctest: +REMOTE_DATA
```

Comment:
For multi-line tests that need remote data, use the directive (
`.. doctest-remote-data::`), here and below (where I don't repeat the suggestion):
```suggestion
.. doctest-remote-data:: 

   >>> from astroquery.solarsystem.pds import RingNode
   >>> import astropy.units as u
   >>> bodytable, ringtable = RingNode.ephemeris(planet='Uranus',
   ...                 epoch='2024-05-08 22:39',
   ...                 location = (-67.755 * u.deg, -23.029 * u.deg, 5000 * u.m))
   >>> print(ringtable)
```